src/index.css

```
@@ -45,8 +45,15 @@
 }
 
 @theme {
-  --font-sans: system-ui, 'Segoe UI', Roboto, sans-serif;
-  --font-mono: ui-monospace, Consolas, monospace;
+  --font-sans: 'IBM Plex Sans', system-ui, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)